try:
    num = int(input('Enter a number'))
    print(num + 5)
except Exception as e:
    print("Error" , e)

print('Hello')